go/vt/vtgate/planbuilder/window.go

```diff
@@ -57,7 +57,7 @@ func isSingleShardPrimitive(route *engine.Route) bool {
 	if route == nil || route.RoutingParameters == nil {
 		return false
 	}
-	return route.RoutingParameters.Opcode.IsSingleShard()
+	return route.Opcode.IsSingleShard()
```

Comment:
Is this intended to be part of the change - if so, I'm not sure how it's part of VDiff